lib/src/index.ts

```diff
@@ -131,9 +153,14 @@ export async function main() {
     }
   }
 
+  if (packageManager !== 'npm') {
+    const destPath = path.join(process.cwd(), projectName as string);
+    await removeFile('package-lock.json', destPath);
```

Comment:
The `core` repository has a `package-lock.json` file, it is OK if the user selects NPM as the package manager, but if they select a non-NPM manager, we need to remove it to avoid conflicts.